apps/cowswap-frontend/src/modules/trade/types/TradeDerivedState.ts

```diff
@@ -17,6 +17,7 @@ export interface TradeDerivedState {
   readonly orderKind: OrderKind
   readonly slippage: Percent | null
   readonly tradeType: TradeType | null
+  readonly supportBuyOrders: boolean
```

Comment:
Added a flag to indicate when trading widget support buy orders (now it's only Swap)